### ISSUE TITLE
Completed Job Log should just be Job Log

### ISSUE THREAD

**Issue by pconrad** [project-anacapa/anacapa-github-linker#109]
And we should add a record to it before starting the job, with a field for "job started" and the "job completed" field null, and summary as "in progress".

Then, when the job completes, update the "job completed" time, and update the "summary".

It might be nice to have a field also that is "how long has it been since the job started" in some kind of user friendly way, e.g. https://github.com/elgalu/time_ago_in_words

**Comment by ProbablyFaiz** [comment]
Completed in 8b2ae13
